src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -594,24 +593,17 @@ SF10A::cycle()
 		return;
 	}
 
-
-	/* Trigger measurement */
-	if (OK != measure()) {
-		DEVICE_DEBUG("measure error lidar");
-	}
-
```

Comment:
OK, agree with your approach, looks good. Have you tested it yet? Would be good to see some logs.